src/forms/forms.stories.html

```diff
@@ -0,0 +1,134 @@
+<article>
+  <h2 class="title">OpenMRS Forms Elements</h2>
+  <p class="title">
+    There are 3 types of form elements defined in the styleguide
+  </p>
+  <div>
+    <div class="container">
+      <div>
+        <h4 class="title">CheckBox</h4>
+        <form>
+          <div class="omrs-input-group">
+            <div class="omrs-checkbox">
+              <input type="checkbox" disabled id="checkbox1" />
+              <label for="checkbox1">Inactive checkbox</label>
+            </div>
+            <div class="omrs-checkbox">
+              <input type="checkbox" disabled checked="true" id="checkbox2" />
+              <label for="checkbox2">Active checkbox</label>
+            </div>
+          </div>
+        </form>
+        <h4 class="title">Radio Button</h4>
+        <form>
+          <div class="omrs-input-group">
+            <div class="omrs-radio-button">
+              <input
+                type="radio"
+                disabled
+                id="radio1"
+                checked="true"
+                name="radio1"
+                group="radio"
+              />
+              <label for="radio1">Active Radio</label>
+            </div>
+            <div class="omrs-radio-button">
+              <input
+                type="radio"
+                disabled
+                id="radio2"
+                name="radio1"
+                group="radio"
+              />
+              <label for="radio2">Inactive Radio</label>
+            </div>
+          </div>
+        </form>
+      </div>
+    </div>
+
+    <h4>Code example:</h4>
+    <p class="type-demo omrs-bold">
+      <%=determineInputType(inputType)%> input
+    </p>
+    <div class="omrs-input-group">
+      <div class="<%=determinStyle(inputType)%>">
+        <input type="<%=determineInputType(inputType)%>" "<%=isChecked%>"
+        id="inputHtml" />
+        <label htmlFor="inputHtml">
+          <%=inputType%>
+        </label>
+      </div>
+    </div>
+    <div class="examples">
+      <div class="tabs">
+        <div lang="html" class="tab">HTML</div>
+        <div lang="jsx" class="tab">JSX</div>
+      </div>
+      <div class="panels">
+        <div class="panel html">
+          <div class="stories-example lang-html">
+            <div className="omrs-checkbox">
+              <input type="checkbox" id="inputHtmlCheckBox" />
```

Comment:
Inputs usually should have `name` attributes. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#name. Would be good to include the name in the documentation.